directory.json

```diff
@@ -162,7 +140,6 @@
   "[hsmr]":"https://hsmr.cc/spaceapi.json",
   "backspace":"https://status.bckspc.de/spacestatus.php",
   "base48":"https://48.io/status/",
-  "brmlab":"http://nat.brmlab.cz/brmd/brmstatus.json",
```

Comment:
I visited this space a few years ago :) Did you contact them?

Reply:
The website doesn't ring a bell right now, might have been down during that time. Sounds like you have a more direct contact possibility :)